hs_app_timeseries/migrations/custom_data_migration_20160718.py

```diff
@@ -0,0 +1,37 @@
+import logging
+
+from django.db import migrations
+
+from hs_core.hydroshare.utils import resource_modified
+from hs_app_timeseries.models import TimeSeriesResource
+
+
+def delete_extracted_metadata(apps, schema_editor):
+    # For all existing timeseries resources, delete all resource specific metadata.
+    # This way we can keep the resource GUID and allow the resource owners to manually
+    # delete the existing SQlite file and then add a new SQlite file to regenerate
+    # resource specific metadata
+    log = logging.getLogger()
+    for res in TimeSeriesResource.objects.all():
+        # delete all extracted metadata
+        res.metadata.sites.delete()
+        res.metadata.variables.delete()
+        res.metadata.methods.delete()
+        res.metadata.processing_levels.delete()
+        res.metadata.time_series_results.delete()
+        message = "Deleted all resource specific metadata from TimeSeries Resource ID:{}"
```

Comment:
I'd be more concerned about weirdness from taking out a migration than I am about leaving such a destructive migration in.